### ISSUE TITLE
Updates to https://github.com/ubam/ubfr-docs/internal/Quick_Links.md

### ISSUE THREAD

**Issue by Acid-Override** [ubam/ubfr-docs#24]
Wayne(out there), [17.02.18 10:03]
I continually receive feedback from people that the 'script' is unnatural and that the word 'welcoming team' is weird because 'everyone in community should welcome - not just  a team' and blah blah blah.  So, I have an easy solution.

Wayne(out there), [17.02.18 10:03]
which is actually easier for us:

Wayne(out there), [17.02.18 10:04]
1. Welcome __!
2. Some custom line of your choice that comes from your heart
3. Check out this ____ (adjective) link to help you get started: (welcome link)

Wayne(out there), [17.02.18 10:05]
in other words:

1. don't use the same script each time (seems ilke spam) and
2. don't use words 'welcoming team'

And we should never hear another word about this! :)
